src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -435,33 +452,65 @@ export default class WebGLRenderer extends SystemRenderer
      * @param {number} location - the texture location
      * @return {PIXI.WebGLRenderer} Returns itself.
      */
-    bindTexture(texture, location = 0)
+    bindTexture(texture, location)
     {
+        location = location || 0;
```

Comment:
Its faster than the babel output. This is a real hot piece of code and figured every little helps :)